.ci/jobs/apm-agent-python-mbp.yml

```diff
@@ -19,7 +19,7 @@
         repo: apm-agent-python
         repo-owner: elastic
         credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
-        head-filter-regex: '^(main|PR-.*|[5-9]\.x|v[0-9].*)$'
```

Comment:
This will make the jenkins jobs only run on the tag event and will be omitted on main and pull requests.